mail/notmuch/Portfile

```diff
@@ -5,10 +5,11 @@ PortGroup           conflicts_build 1.0
 
 name                notmuch
 version             0.28.3
+revision            1
```

Comment:
The `revision` does not need to be changed. Please set it back to 0.

Reply:
Done.